content/getting-started/quickstart.textile

```diff
@@ -407,7 +407,7 @@ ARTRealtime *ably = [[ARTRealtime alloc] initWithKey:@"{{API_KEY}}"];
 ```
 
 ```[flutter]
-final clientOptions = ably.ClientOptions.fromKey('{{API_KEY}}');
+final clientOptions = ably.ClientOptions(key: '<KEY>');
```

Comment:
If you keep the `'{{API_KEY}}'` syntax instead of `'<KEY>'`  then this is what replaces the key with a demo API key when the page renders.